plover/dictionary/rtfcre_dict.py

```diff
@@ -335,13 +336,23 @@ def save_dictionary(d, fp):
     writer = codecs.getwriter('cp1252')(fp)
     writer.write(HEADER)
 
+    error_msg = ''
     for s, t in d.items():
         s = '/'.join(s)
         t = format_translation(t)
         entry = "{\\*\\cxs %s}%s\r\n" % (s, t)
-        writer.write(entry)
+        try:
+            writer.write(entry)
+        except Exception:
+            error_msg = error_msg + entry + "\n"
```

Comment:
Here, no need to catch Exception. So far the only error we've seen is:

`except UnicodeEncodeError:`

For the errors, I'd probably want some more info as a developer than just the string. Perhaps a description that we can display to the user would be useful?

So `failed_entries.append((entry, _('Plover does not support Unicode characters in RTF`))` ?

Then when you are compiling the QDialog you can use this list instead of just a string.